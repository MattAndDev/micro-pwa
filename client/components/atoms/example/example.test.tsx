import { h } from 'preact'
import { render } from '@testing-library/preact'
import { Example } from './example'

it('renders correctly', () => {
  const { container } = render(<Example type="button">Oh hi!</Example>)
  expect(container.firstChild).toMatchSnapshot()
})
