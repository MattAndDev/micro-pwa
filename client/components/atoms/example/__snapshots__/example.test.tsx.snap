// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  class="wrap"
  type="button"
>
  Oh hi!
  aa
</button>
`;
